db-engine/src/main/java/com/splicemachine/db/iapi/sql/execute/ExecRow.java

```diff
@@ -111,6 +111,8 @@
 
 	StructType createStructType();
 
+	StructType createStructTypeCorrected(int[] baseColumnMap);
```

Comment:
createStructType() isn't used anywhere else, right? You should just overwrite it, not add a new method.